libparsec/crates/protocol/schema/authenticated_cmds/shamir_recovery_setup.json5

```diff
@@ -51,7 +51,14 @@
             },
             {
                 // Future evolution 1: Shamir recovery has already been setup, should ask your admin to reset it first !
-                "status": "shamir_setup_already_exists"
+                "status": "shamir_setup_already_exists",
+                "fields": [
+                    {
+                        "name": "last_shamir_certificate_timestamp",
+                        "type": "DateTime"
+                    }
+
+                ]
```

Comment:
Is this implemented? I'm asking because the comment before says "Future evolution 1..."

Reply:
Yes it is implemented, I'm removing the future evolution comment [here](https://github.com/Scille/parsec-cloud/pull/7609#discussion_r1647292959)